packages/app/src/cli/commands/app/config/link.ts

```diff
@@ -34,14 +33,15 @@ export default class ConfigLink extends AppCommand {
       apiKey: flags['client-id'],
     }
 
-    await link(options)
+    const result = await link(options)
 
-    const specifications = await loadLocalExtensionsSpecifications()
-
-    const app = await loadApp({
-      specifications,
+    const {app} = await linkedAppContext({
       directory: flags.path,
-      userProvidedConfigName: undefined,
+      clientId: undefined,
+      forceRelink: false,
+      userProvidedConfigName: result.state.configurationFileName,
+      // Using report because a bad extension config shouldn't prevent the app from being linked
+      mode: 'report',
```

Comment:
For all of these -- is this `mode` change accurate? Should it be? This feels like a more flexible context function. We should keep this surface as tight as we can and resist making it flexible. Like, maybe `env show` could work if your app is broken, but _should_ it?

Reply:
Updated, left the report mode only for `info`.